### PR TITLE
CMCL-1457: Remove test framework from cinemachine package

### PR DESCRIPTION
**Pull request by AntoineCharton**
[Delete any line or section that does not apply]

### Purpose of this PR

Remove dependancy to test framework

### Testing status

- [ ] Added an automated test
- [ ] Passed all automated tests
- [ ] Manually tested 

### Technical risk

Medium testing changes. Normally no changes in the package though need some manual testing just to be sure.

### Comments to reviewers

Check everything is tested correctly. The number of file changed look scary but most of it is just the test folder being moved to an other package. 

### Package version

- [ ] Updated package version

### CODE REVIEW

File: Projects/HDRP/Packages/manifest.json [hunk 1]
```diff
@@ -47,6 +48,7 @@
   },
   "testables": [
     "com.unity.cinemachine",
```
Comment (glabute) [inline Projects/HDRP/Packages/manifest.json:50]: is Cinemachine still a testable?

File: Projects/HDRPInputSystem/Packages/manifest.json [hunk 2]
```diff
@@ -47,6 +48,7 @@
   },
   "testables": [
     "com.unity.cinemachine",
```
Comment (glabute) [inline Projects/HDRPInputSystem/Packages/manifest.json:50]: still need this?

File: Projects/Standalone/Packages/manifest.json [hunk 3]
```diff
@@ -41,6 +42,7 @@
     "com.unity.modules.xr": "1.0.0"
   },
   "testables": [
-    "com.unity.cinemachine"
+    "com.unity.cinemachine",
```
Comment (glabute) [inline Projects/Standalone/Packages/manifest.json:45]: same

File: Projects/StandaloneInputSystem/Packages/manifest.json [hunk 4]
```diff
@@ -42,6 +43,7 @@
     "com.unity.modules.xr": "1.0.0"
   },
   "testables": [
-    "com.unity.cinemachine"
```
Comment (glabute) [inline Projects/StandaloneInputSystem/Packages/manifest.json:45]: meme chose

File: Projects/URP/Packages/manifest.json [hunk 5]
```diff
@@ -45,6 +46,7 @@
     "com.unity.modules.xr": "1.0.0"
   },
   "testables": [
-    "com.unity.cinemachine"
+    "com.unity.cinemachine",
```
Comment (glabute) [inline Projects/URP/Packages/manifest.json:49]: anche questo

File: Projects/URPInputSystem/Packages/manifest.json [hunk 6]
```diff
@@ -46,6 +47,7 @@
     "com.unity.modules.xr": "1.0.0"
   },
   "testables": [
-    "com.unity.cinemachine"
+    "com.unity.cinemachine",
```
Comment (glabute) [inline Projects/URPInputSystem/Packages/manifest.json:50]: this too

**Review by glabute — CHANGES_REQUESTED** [review]
will try it locally... stay tuned

**Review by glabute — COMMENTED** [review]
This is great, I love it

File: Projects/HDRPInputSystem/Packages/manifest.json (hunk 2 above)
Comment (AntoineCharton) [reply Projects/HDRPInputSystem/Packages/manifest.json:50]: Hmm good point. May not be needed. Will try.

Comment (AntoineCharton) [reply Projects/HDRPInputSystem/Packages/manifest.json]: Works locally. Gonna test in Yamato but isn't needed apparently. Nice catch!

File: com.unity.cinemachine.tests/package.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "com.unity.cinemachine.tests",
+  "type": "tests",
+  "version": "1.0.0",
+  "displayName": "Cinemachine Tests",
+  "dependencies": {
+    "com.unity.test-framework": "1.1.33"
```
Comment (glabute) [inline com.unity.cinemachine.tests/package.json:7]: don't we also depend on cinemachine itself?

Comment (AntoineCharton) [reply com.unity.cinemachine.tests/package.json:7]: According to the doc that is not needed but will check.

**Comment by AntoineCharton** [comment]
Will probably need @windxu88 input to solve this issue would be great to solve for 3.1.X.

**Pull request closed without merging**